en/components/accordion.md

```diff
@@ -0,0 +1,264 @@
+
+<code-view style="height:550px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/layouts/accordion-sample-2" alt="Angular Avatar Example">
+</code-view>
+
+<div class="divider--half"></div>
+
+## Keyboard Navigation
+Keyboard navigation in the accordion provides a rich variety of keyboard interactions for the user. This functionality is enabled by default and allows users to easily navigate through the panels.
+The [IgxAccordionComponent]({environment:angularApiUrl}/classes/igxaccordioncomponent.html}) navigation is compliant with W3C accessibility standards and convenient to use.
+
+**Key Combinations**
+
+ - <kbd>Tab</kbd> - moves the focus to the first(if the focus is before accordion)/next panel
+ - <kbd>Shift + Tab</kbd> - moves the focus to the last(if the focus is after accordion)/previous panel
+ - <kbd>Arrow Down</kbd> - moves the focus to the panel below
+ - <kbd>Arrow Up</kbd> - moves the focus to the panel above
+ - <kbd>Alt + Arrow Down</kbd> - expands the focused panel in the accordion
+ - <kbd>Alt + Arrow Up</kbd> - collapses the focused panel in the accordion
+ - <kbd>Shift + Alt + Arrow Down</kbd> - expands all enabled panels(if singleBranchExpand is set to true expands the last enabled panel)
+ - <kbd>Shift + Alt + Arrow Up</kbd> - collapses all enabled panels
+ - <kbd>Home</kbd> - navigates to the FIRST enabled panel in the accordion
+ - <kbd>End</kbd> - navigates to the LAST enabled panel in the accordion
+
+## Angular Accordion Styling
+The [accordion]({environment:angularApiUrl}/classes/igxaccordioncomponent.html}) serves only as a container for the underlying [panels]({environment:angularApiUrl}/classes/igxexpansionpanelcomponent.html). Styles can be applied directly through the panel's theme, as described in the [styling section of the IgxExpansionPanel topic](expansion-panel.html#styling).
+
+By design, there is a margin set to the expanded panels, in case that they are positioned within an `igx-accordion`. In order to modify it there is a property exposed inside the igx-expansion-panel theme.
+In order to take advantage of the functions exposed by the theme engine, we have to import the `index` file in our style file:
+
+```scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+```
+
+Following the simplest approach, we create a new theme that extends the [igx-expansion-panel-theme]({environment:sassApiUrl}/index.html#function-igx-expansion-panel-theme) and accepts the `$expanded-margin`   parameter. 
+```scss
+$custom-panel-theme: igx-expansion-panel-theme(
+    $expanded-margin: 0px
+);
+```
+### Using CSS Variables
+
+The last step is to include the component's theme.
+```scss
+:host {
+    @include igx-css-vars($custom-panel-theme);
+}
+```
+
+### Using Theme Overrides
+
+In order to style components for older browsers, like Internet Explorer 11, we have to use a different approach, since CSS variables are not supported there.
```

Comment:
```suggestion
In order to style components in Internet Explorer 11, we have to use a different approach, since CSS variables are not supported there.
```